src/doc/using_visit/Plots/PlotTypes/VolumePlot.rst

```diff
@@ -54,6 +54,10 @@ Each sample point along the ray is composited to form a final color for the scre
 Rays are traced from closest to farthest to allow for early ray termination which stops the sampling process when the pixel opacity gets above a certain threshold.
 This method of volume-rendering yields superior pictures at the cost of speed and memory use.
 
+If VisIt_ is built with `ANARI <https://www.khronos.org/api/index_2017/anari/>`_ support by using the ``-–anari`` option, 
```

Comment:
The style standard for our RST is one sentence per line. Would you please fix for the lines you have added.